plugins/firehydrant/src/api/index.ts

```diff
@@ -77,10 +78,12 @@ export class FireHydrantAPIClient implements FireHydrantAPI {
 
   async getServiceDetails(options: {
     serviceName: string;
+    lookupByName: boolean;
   }): Promise<ServiceDetailsResponse> {
+    const queryOpt = options.lookupByName ? 'name' : 'query';
     const proxyUrl = await this.getApiUrl();
     const response = await fetch(
-      `${proxyUrl}/services?query=${options.serviceName}`,
+      `${proxyUrl}/services?${queryOpt}=${options.serviceName}`,
```

Comment:
While we're here, should probably `encodeURIComponent` the value to make it URL safe. Or even better, do

```
const query = new URLSearchParams();
query.set(queryOpt, options.serviceName);
// ...
/services?${query}
```

so it can do the encoding as best it wants.

Reply:
Seems reasonable.